### PR TITLE
Week 2 assignments

### PR DESCRIPTION
**Pull request by sachin286**

### CODE REVIEW

File: Week-2/Exercise-2.2/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/index.html:1]: Please use a Prettifer plugin to intend the code.

File: Week-2/Exercise-2.2/style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+*{
+	margin:0px;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Same here for indentation.

File: Week-2/Exercise-2.2/style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	padding:0px;
+	list-style-type:none;
+}
+
+body{
+	margin: 0;
+	padding: 10px 80px 10px 80px;
+	background-size: cover;
+	font-family: Arial;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Please add this as variable

File: Week-2/Exercise-2.2/style.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+	float: left;
+}
+
+.right_section {
+	margin-top: 10px;
+	width: 46%;
+	float: right;
+}
+
+@media only screen and (max-width: 629px) {
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Please add media queries to the end of the file

File: Week-2/Exercise-2.2/style.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.button {
+  margin-top:25px;
+  padding: 15px 25px;
+  font-size: 24px;
+  text-align: center;
+  cursor: pointer;
+  outline: none;
+  color: #fff;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.2/style.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.field{
+	margin-top:25px;
+}
+
+.button {
+  margin-top:25px;
+  padding: 15px 25px;
+  font-size: 24px;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.2/style.css [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  border: none;
+  border-radius: 15px;
+  box-shadow: 0 9px #999;
+}
+
+.button:hover {background-color: #3e8e41}
+
+.button:active {
+  background-color: #3e8e41;
+  box-shadow: 0 5px #666;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.2/style.css [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  cursor: pointer;
+  outline: none;
+  color: #fff;
+  background-color: #04AA6D;
+  border: none;
+  border-radius: 15px;
+  box-shadow: 0 9px #999;
+}
+
+.button:hover {background-color: #3e8e41}
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.2/style.css [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+.button {
+  margin-top:25px;
+  padding: 15px 25px;
+  font-size: 24px;
+  text-align: center;
+  cursor: pointer;
+  outline: none;
+  color: #fff;
+  background-color: #04AA6D;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.2/style.css [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+  border: none;
+  border-radius: 15px;
+  box-shadow: 0 9px #999;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.2/style.css [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.button:hover {background-color: #3e8e41}
+
+.button:active {
+  background-color: #3e8e41;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: Move this to variables please.

File: Week-2/Exercise-2.1/style.css [hunk 12]
```diff
@@ -0,0 +1,86 @@
+:root {
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.1/style.css:1]: Good work on the variables.

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by sachin286** [comment]
I have updated the files on the variables and styling

File: Week-2/Exercise-2.2/style.css [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+  --red: #ff0000;
+  --yellow: #ffff00;
+  --blue: #1d0eed;
+  --green: #3e8e41;
+  --light-green: #3e8e41;
+  --medium-green: #04aa6d;
+  --grey: #999;
+
+  --text-size: 24px;
+  --text-fam: Arial;
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.2/style.css]: This can be renamed to `--font-family`

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by sachin286** [comment]
variables names updated

**Comment by ShashankSuresh** [comment]
1. Adding meta tags.
2. Responsiveness for mobile, tablet and desktop devices. 
3. Formatting (minor)
4. Attach screen-shots 
5. Beautification of form

File: Week-2/Exercise-2.1/style.css [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+
+.main-wrapper:hover .div-four {
+    transform: rotate(90deg) translateY(-65px);
+    transition: all ease 1s;
+}
+
+.main-wrapper:hover .div-two {
+    transform: rotate(90deg) translateY(-65px);
+    transition: all ease 1s;
+}
```
Comment (ShashankSuresh) [inline Week-2/Exercise-2.1/style.css:86]: The rotation for Blue and Yellow is reversed. Kindly made the necessary modification.

**Comment by sachin286** [comment]
Screenshot 2022-07-01 at 7 22 24 PM

updated the format and added the changes mentioned

**Review by ShashankSuresh — APPROVED** [review]

**Pull request merged**